test/src/main/java/org/axonframework/test/aggregate/Reporter.java

```diff
@@ -194,6 +195,26 @@ public void reportWrongExceptionMessage(Throwable actualException, Description d
                                              NEWLINE);
     }
 
+    /**
+     * Report an error due to a difference in exception details.
+     *
+     * @param actualException The actual exception
+     * @param description     A description describing the expected value
+     */
+    public void reportWrongExceptionDetails(Throwable actualException, Description description) {
```

Comment:
Wouldn't it be more reasonable to provide the actual exception's details, instead of the entire actual exception?

Reply:
I agree it makes more sense, I changed this